keps/sig-network/563-dual-stack/README.md

```diff
@@ -1392,20 +1388,18 @@ This capability will move to stable when the following criteria have been met.
   N/A
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  We did manual testing of a cluster turning it off and on to explore
+  disabled-with-data behavior.
+  [TODO] Describe outcome of testing
```

Comment:
There was a request for more detail about the test outcomes. Can we describe that?

Reply:
Testing here: https://github.com/kubernetes/kubernetes/blob/master/test/integration/dualstack/dualstack_test.go